### ISSUE TITLE
Unable to view table in Psequel in macOS Mojave v.10.14.5

### ISSUE THREAD

**Issue by aleph1999** [psequel/psequel#206]
I have been able to write a sample table, and am able to query for it when needed, but am still unable to view the tables in my Psequel home screens.

**Comment by jagillan** [comment]
The same issue is happening with me.

**Comment by mleers** [comment]
Cannot view tables or use search function after refreshing in High Sierra (10.13.6).  Rows are acknowledged in gui under content tab, as well as in command line \dt.

**Comment by mk5559** [comment]
same with mac os Catalina 10.15. I can't view tables. 
![image](https://user-images.githubusercontent.com/33440900/71332871-ea0ce800-2505-11ea-8aa0-c0cc102227bb.png)

**Comment by tkbky** [comment]
I'm having the same issue when I'm connecting it to postgresql 12, but it works fine with postgresql 11.

**Comment by haasman** [comment]
Mac OS Catalina 10.15.2 Is not showing the rows for me. Postgresql 12.

**Comment by shusson** [comment]
Mac OS Catalina 10.15.2 

psequel was working on postgresql 11 but stopped working when I changed to postgres 12

**Comment by lyonsun** [comment]
Same here.

Mac OS Catalina 10.15.2 with postgres (PostgreSQL) 12.1.

**Comment by dylangit01** [comment]
same issue here, not able to see the rows

**Comment by coreyladovsky** [comment]
Many of my students are having the same issue.

**Comment by bojanpopic** [comment]
Same here. Listings are empty since we switched to postgres 12.

**Comment by ali92hm** [comment]
Seems like this is an issue with PS 12 in general since people on multiple versions of MacOS are facing it. 

I'm on MacOS Catalina (10.15.3) and tried this with both with PS 11 and PS 12. Everything works fine with PS 11 but switching to PS 12 makes the rows not show up.

**Comment by NSCabezon** [comment]
is it a bug from PS? can't it be fixed in PSequel?

**Comment by nitinnain** [comment]
Just upgraded to Postgres 12 and it doesn't show the table data (The issue is not for MacOS version but psql version.)

**Comment by martinburger** [comment]
This seems to be caused by the following error:

```
2020-05-26 13:27:27.299 CEST [52321] ERROR:  column c.relhasoids does not exist at character 245
```

According to [this answer at Stack Overflow](https://stackoverflow.com/a/58461430), the issue "is because in v. 12, table OIDs are no longer treated as special columns, and hence the relhasoids column is no longer necessary."

**Comment by idoco** [comment]
@martinburger do we have to source code to fix this issue? I'm willing to give it a try.

Also, I suggest renaming this issue to "Unable to view PostgreSQL 12 tables" or something like that.

**Comment by martinburger** [comment]
> do we have to source code to fix this issue? I'm willing to give it a try.

Yes, I think one has to fix this issue in the source code. It is not publicly available, is it?

**Comment by ZormeneKinanee1** [comment]
I'm experiencing the same issue with PSequel and so I just switched to TablePlus. Here is the link to download it. Hope this helps! 
https://tableplus.com/download

**Comment by paulbhartzog** [comment]
> I'm experiencing the same issue with PSequel and so I just switched to TablePlus. Here is the link to download it. ...

@ZormeneKinanee1

It's pretty bad form to recommend an alternative on an issue for a program.  This Issue exists to help the developer fix things and thereby benefit the community.  Please be courteous and respectful to the community for this program.

**Comment by paulbhartzog** [comment]
me = Mojave, Postgres 12
I get alternate colored rows like the screenshot above, but no data on the Content tab. :-)

**Comment by doughsay** [comment]
> It's pretty bad form to recommend an alternative on an issue for a program. This Issue exists to help the developer fix things

This project is, for all intents and purposes, dead.  Alternatives have been suggested in many other threads.  There has not been a release since 2017, and the twitter account has not been active since then either.  The source code is not available, therefore, nobody can help fix the many issues it has.  Suggesting alternatives is the only recourse we have now...

FWIW, I and all my work colleagues have also switched to TablePlus, although it should be pointed out it is not free...

**Comment by paulbhartzog** [comment]
> This project is, for all intents and purposes, dead. Alternatives have been suggested in many other threads. There has not been a release since 2017, and the twitter account has not been active since then either. The source code is not available, therefore, nobody can help fix the many issues it has. Suggesting alternatives is the only recourse we have now...
> 
> FWIW, I and all my work colleagues have also switched to TablePlus, although it should be pointed out it is not free...

@doughsay Thanks for the reply and info!

**Comment by doughsay** [comment]
You're welcome, but also, I should have pointed out that your sentiment is usually absolutely correct.  You shouldn't promote alternative software in github issues in general.  I just make an exception in the case of abandoned projects. :wink:

**Comment by nitinnain** [comment]
Such a shame to see this product go. It could've become so much more if it was open source.

Btw, PSequel seems to have taken its inspiration from the open source project https://www.sequelpro.com/ (which is for MySQL only).
